language/move-prover/bytecode-to-boogie/test_mvir/verify-stdlib/libra_account.mvir

```diff
@@ -114,6 +118,8 @@ module LibraAccount {
         aborts_if !global_exists<Self.T>(payee)
         aborts_if to_deposit.value == 0
         aborts_if global<Self.T>(payee).balance.value + to_deposit.value > max_balance()
+        aborts_if global<Self.T>(txn_sender).sent_events.counter + 1 > max_balance()
```

Comment:
I'm not sure where max_balance() is defined.  I see it in libra_account.prover.bpl, but I don't know how it got in there.

U64_MAX is defined in the prelude and seems to be the same number.

Comparing the events counter with max_balance is confusing, because the events counter has nothing to do with account balances.  Comparing with U64_MAX would be more meaningful.

Reply:
I'm going to define a max_u64() in prelude.bpl and just use max_64() instead of max_balance() everywhere.